bugs/PENDING_SUBMIT/0397-BAD-IMAGE-S20PLUS-41991e5f-no-opt-test-SM-G986B-R3CNA03E21W/reduced_manual/variant/shader.frag

```diff
@@ -0,0 +1,122 @@
+#version 320 es
+
+precision highp int;
+
+precision highp float;
+
+// Contents of injectionSwitch: [0.0, 1.0]
+layout(set = 0, binding = 0) uniform buf0 {
+ highp vec2 injectionSwitch;
+};
+precision highp float;
+
+precision highp int;
+
+struct struct_base {
+ int data;
+ int leftIndex;
+ int rightIndex;
+} ;
+
+struct_base struct_a[10] = struct_base[10](
+                            struct_base(1, - 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1)
+                            );
+
+void func_a(inout struct_base struct_a, int input_a)
+{
+ struct_a.data = 1;
+}
+void func_b(int input_b, int input_a)
+{
+ int var_a = 0;
+ int var_b = 0;
+ // Always true. Iterates 6 times,
+ // but returns nothing on first iteration.
+ while(1 <= 1)
+  {
+   if(var_b >= 5)
+    {
+     break;
+    }
+   var_b ++;
+   // Clamp returns 0. Struct_a[0].leftIndex == -1.
+   // Always true.
+   if(struct_a[clamp(var_a, 0, 10 - 1)].leftIndex == - 1)
+    {
+     func_a(struct_a[1], 1);
+     return;
+    }
+   else
+   // this is never reached
+    {
+     var_a = 1;
+     continue;
+    }
+  }
+}
+precision highp float;
+
+precision highp int;
+
+layout(location = 0) out vec4 _GLF_color;
+
+// Contents of resolution: [256.0, 256.0]
+layout(set = 0, binding = 1) uniform buf1 {
+ vec2 resolution;
+};
+float compute_value(float limit, float thirty_two)
+{
+ float result = - 0.5;
+ for(int i = 1; i < 800; i ++)
+  {
+   if((i % 32) == 0)
+    {
+     result += 0.4;
+    }
+   else
+    {
+     if(mod(float(i), round(thirty_two)) <= 0.01)
+      {
+       result += 100.0;
+      }
+    }
+   // This function has no effect to the rest of the shader.
+   // Variant shader is semantically same as reference shader.
```

Comment:
effect to -> effect on
And I think there is no need for the semantically equivalent comment again (as we have it in the AmberScript). So just:

// This function call does not affect the output of the shader.